nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
 
     def model_provider_func(self, pre_process, post_process):
         cfg = self.cfg
-        num_tokentypes = 2 if cfg.bert_binary_head else 0
+        num_tokentypes = self.cfg.get('num_tokentypes', 2)
```

Comment:
Can you add an assertion here saying if bert binary head is true, and num token types is not true its a mistake.